### DISCUSSION TITLE
Contrast agents fMRI

### DISCUSSION THREAD

**Discussion by SjoerdMurris** [PRIME-RE/prime-re.github.io#83] [General]
Hi everyone,

We're planning to start fMRI experiments soon-ish here in Amsterdam. We previously have only done BOLD scans here, but I've done CBV scans before in Leuven, injecting a contrast agent (MION) before scanning. We have an old batch of Feraheme lying around, which has expired. We're a bit hesitant in using it, since Feraheme is advising against it (even after emphasizing it's not for clinical purposes). I was always under the impression that these contrast agents do not expire, but something to do with particle size and microbiological purity, who knows..

In any case, we're thinking of ordering new vials through BioPal, which I know they also order from at my old group in Leuven. But before I do so, I thought it would be nice to reach out and see what other people are using these days?

All the best,
Sjoerd

**Comment by PRIME-DRE** [comment]
Hi Sjoerd,

We use BioPal as well (and also used to use Feraheme). 

Best,

~Theresa

—
Theresa M. Desrochers, Ph.D.
Rosenberg Assistant Professor of Brain Science
Department of Neuroscience
Department of Psychiatry and Human Behavior (courtesy)
Carney Institute for Brain Science
Brown University
http://neuroscience.brown.edu/desrochers

**Reply by SjoerdMurris** [reply]
Great! Thanks for letting me know Theresa!

**Comment by PRIME-DRE** [comment]
You may want to check with your vets. Here in the US it is not permitted to use expired drugs in monkeys or any animals .Michele On Jan 12, 2024, at 3:48 AM, 'SjoerdMurris' via NHP-MRI ***@***.***> wrote:﻿
Hi everyone,
We're planning to start fMRI experiments soon-ish here in Amsterdam. We previously have only done BOLD scans here, but I've done CBV scans before in Leuven, injecting a contrast agent (MION) before scanning. We have an old batch of Feraheme lying around, which has expired. We're a bit hesitant in using it, since Feraheme is advising against it (even after emphasizing it's not for clinical purposes). I was always under the impression that these contrast agents do not expire, but something to do with particle size and microbiological purity, who knows..
In any case, we're thinking of ordering new vials through BioPal, which I know they also order from at my old group in Leuven. But before I do so, I thought it would be nice to reach out and see what other people are using these days?
All the best,
Sjoerd

—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>




-- 
You received this message because you are subscribed to the Google Groups "NHP-MRI" group.
To unsubscribe from this group and stop receiving emails from it, send an email to ***@***.***
To view this discussion on the web visit https://groups.google.com/d/msgid/nhp-mri/PRIME-RE/prime-re.github.io/repo-discussions/83%40github.com.

**Reply by SjoerdMurris** [reply]
Thanks Michele, that makes sense and I'll check here as well with our vets/animal caretakers. It's interesting that BioPal states there's no expiry date though, whereas Feraheme does (and they sell very comparable products). So I'm thinking it's more of a legal rather than a safety issue. Nonetheless, important to check, thanks!

**Comment by beruss** [comment]
Hi Sjeord, 

As with Theresa, I've used both Feraheme and MOLDAY.  my best understanding is that because Feraheme is/was designed as an clinical iron supplement not just a contrast agent it more easily binds to tissue and sticks around. I have seen this first hand where we see significant dropout after only a few sessions with Feraheme, whereas MOLDAY appears to clear much more quickly and thoroughly. If you have the option to switch to Biopal's MOLDAY I would recommend it. Otherwise you may want to pay even greater attention to changes in signal across sessions.

**Reply by SjoerdMurris** [reply]
Hi Brian,

Thanks for your response, very helpful! Certainly something to take into account. I'm assuming here that you used a similar chelation therapy protocol post-scanning? Are you referring to a drop in SNR (mean signal in the brain, divided by the variance of the signal outside of the brain; or potentially some other measure)?

**Reply by beruss** [reply]
hi Sjoerd,
We've changed our chelation protocols over time, but for the comparison I was talking about they were basically without chelation in both cases.  Yes drop in SNR, but not just the mean but localized drops particularly in the anterior temporal lobes, medial orbital and the top of the cortex.

**Reply by SjoerdMurris** [reply]
Hi Brian,

Good to know! I don't think anyone has ever quantified the differences in signal and metabolism between Feraheme and MION for NHPs (not that I'm aware of in any case). So this is certainly helpful. I suppose coil placement is the same here. We'll take it into account for our experiments and in our considerations whether we'll use Feraheme and/or MION.
